book_ddl.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE hafifa.unprocessed_books (
+    isbn VARCHAR(20) PRIMARY KEY,
+    book_title VARCHAR(255),
+    book_author VARCHAR(255),
+    year_of_publication INTEGER,
+    publisher VARCHAR(255)
+);
+
+CREATE TABLE hafifa.book (
+    book_id SERIAL PRIMARY KEY,
+    book_name VARCHAR(255)
+);
```

Comment:
Maybe you should try using `sqlalchemy` instead of writing all of the queries and such like this. (Also in the code you write queries using strings, which is okay, but consider it)

Reply:
I agree and have decided to shift to sql alchemy.
it is ORM, but sync. meaning there would be some additional work
in making it async. postponed for now for that reason.